### PR TITLE
Connect to devnet: NFT feature stanza

### PR DESCRIPTION
**Pull request by mDuo13**
- Updates the required `[features]` stanza for connecting to Devnet to have the NFT feature that's expected there soon
- Removes $ from shell commands to facilitate copy-pasting

### CODE REVIEW

**Comment by manojsdoshi** [comment]
LGTM

**Review by manojsdoshi — APPROVED** [review]
LGTM

**Review by intelliot — APPROVED** [review]

**Comment by tequdev** [comment]
https://github.com/XRPLF/xrpl-dev-portal/blob/361209e027c758f60e81535f0ae5e50b79b27b00/content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md?plain=1#L46

Can you please fix this ordered list ?
<br>
```
3. Comment out the previous `[ips]` stanza, if there is one:
```
to

```
1. Comment out the previous `[ips]` stanza, if there is one:
```

**Review by manojsdoshi — APPROVED** [review]
LGTM

**Comment by mDuo13** [comment]
Updated:

- NFTs will go to their own network ("NFT-Devnet") not the current Devnet. Added the relevant details in a lot of places.
- Fixed #1250.
- Fixed the numbering typo @develoQ pointed out.

- [ ] TODO: UNL address & keys for the NFT-Devnet. @manojsdoshi is working on that over the weekend.

File: content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md [hunk 1]
```diff
@@ -83,8 +117,18 @@ Edit your `validators.txt` file. This file is located in the same folder as your
         [validator_list_keys]
         ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
 
+    *NFT-Devnet*
+
+        [validator_list_sites]
+        TODO
+
+        [validator_list_keys]
+        TODO
```
Comment (mDuo13) [inline content/tutorials/manage-the-rippled-server/configuration/connect-your-rippled-to-the-xrp-test-net.md]: Note: Make these not "TODO" before merging

**Pull request merged**